config/application.rb

```diff
@@ -1,6 +1,7 @@
 require_relative 'boot'
 
 require 'rails/all'
+require 'csv'
```

Comment:
Is this nessecary now that we're doing bulk upload by pictures?

Reply:
No, it is not. I will remove it.